File: webserver.py
from flask import Flask

app = Flask(__name__)
app.config['DEBUG'] = True


@app.route('/')
def hello_world():
    message = 'Hello World. Welcome to the Flask Dev Server.' + \
              '<br/>Created By: Dan Sheffner'
    return message

if __name__ == "__main__":
    app.run(host='0.0.0.0', port=8000)

File: webserver.go
/*
Simple web server created by Dan Sheffner
*/

package main

import (
	"fmt"
	"net/http"
)

func hello_world(w http.ResponseWriter, req *http.Request) {
	w.Write([]byte("Hello World. Welcome to Go Web Server.\nCreated: By Dan Sheffner"))
}

func main() {
	fmt.Println("Starting web server...")
	http.HandleFunc("/", hello_world)
	http.ListenAndServe(":8080", nil)
}
